lib/html-proofer/log.rb

```diff
@@ -5,16 +5,23 @@ module HTMLProofer
   class Log
     include Yell::Loggable
 
-    def initialize(log_level)
-      @logger = Yell.new(format: false, \
-                         name: 'HTMLProofer', \
-                         level: "gte.#{log_level}") do |l|
-        l.adapter :stdout, level: %i[debug info warn]
-        l.adapter :stderr, level: %i[error fatal]
+
+    def initialize(log_level, hide)
+      if !hide
```

Comment:
Instead of this, please do

```
 @logger = HTMLProofer::Log.new(@options[:log_level]) unless @options[:stat]
``` 

earlier in the code. Every method in this class having to do `return if` seems like a smell.